Business/Concrete/ProductManager.cs

```diff
@@ -23,13 +26,15 @@ public ProductManager(IProductRepository productRepository)
     public async Task<Product> GetByIdAsync(Guid productId) => 
         await _productRepository.GetAsync(entity => entity.Id == productId);
 
-    public async Task<List<Product>> GetListAsync() => 
-        await _productRepository.GetListAsync();
+    public async Task<List<Product>> GetListAsync(PaginationParameters paginationParameters) =>
+        _productRepository.GetListAsync().Result
+            .Skip((paginationParameters.PageNumber - 1) * paginationParameters.PageSize)
```

Comment:
I think you can create auto-property with getter only in `PaginationParameters` to calculate skip amount of items, it will make your approach unified and reusable across other endpoints/services